README.md

```diff
@@ -141,7 +142,7 @@ let started = await cells.start(<runtime.StartExecutableRequest>{
 [spawned aurae instances]: https://aurae.io/stdlib/v0/#instance "Short lived nested virtual instances of Aurae"
 [v0 api reference]: https://aurae.io/stdlib/v0/ "Learn more about the current Aurae library definitions"
 [virtual machines]: https://aurae.io/stdlib/v0/#virtualmachine "Long-lived arbitrary virtual machines"
-[introducing Aurae cells]: https://aurae.io/blog/2022-10-24-aurae-cells/#IntroducingAuraeCells "Aurae Blog: 2022-10-24"
+[introducing aurae cells]: https://aurae.io/blog/2022-10-24-aurae-cells/#IntroducingAuraeCells "Aurae Blog: 2022-10-24"
```

Comment:
Aurae should be capitalised I think.

Reply:
It's not visible, and only for linking purpose. The place where the link "introducing aurae cells" is called and how it is written take priority and will be displayed.

Reply:
I used [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for markdown formatting (except table - those I kept for readability - ).

Reply:
reference: https://github.github.com/gfm/#example-599